common/src/main/java/com/wynntils/commands/WynntilsCommand.java

```diff
@@ -60,8 +60,7 @@ private int version(CommandContext<CommandSourceStack> context) {
     }
 
     private int reload(CommandContext<CommandSourceStack> context) {
-        for (Feature feature :
-                FeatureRegistry.getFeatures().values()) { // disable all active features before resetting web
+        for (Feature feature : FeatureRegistry.getFeatures()) { // disable all active features before resetting web
```

Comment:
Perhaps only disable features that are affected by web stuff (and later other conditions that can be "reloaded")? Although, this is not a simple design change, and conditions should get their own PR imo.

Reply:
Yeah this is currently an oversight. We need a way to distinguish between features that can fail to enable due to web api, etc